lib/core.js

```diff
@@ -330,6 +330,8 @@ type Iterable<T> = $Iterable<T,void,void>;
 interface Generator<Yield,Return,Next> {
     @@iterator(): $Iterator<Yield,Return,Next>;
     next(value?: Next): IteratorResult<Yield,Return>;
+    return(value: Return): IteratorResult<Yield,Return>;
```

Comment:
Could we do something like

```JavaScript
return<R: Return>(value: R): IteratorResult<Yield,R>;
```

So this would allow us to write

```JavaScript
function foo(g: Generator<string, ?string, void>): IteratorResult<string, string> {
  return g.return('hello');
}
```